src/pages/TrueSightV2/pages/News.tsx

```diff
@@ -0,0 +1,185 @@
+import { Text } from 'rebass'
+import styled, { useTheme } from 'styled-components'
+
+const Wrapper = styled.div`
+  padding: 20px 0;
+  width: 100%;
+  display: flex;
+  gap: 24px;
+  flex-wrap: wrap;
+  align-items: stretch;
+  > * {
+    width: calc(33.33% - 24px * 2 / 3);
```

Comment:
```suggestion
    width: calc(33.33% - 16px);
```
Why this is so complicate 😕

Reply:
this file will be remove